packages/babel-jest/src/index.js

```diff
@@ -82,24 +79,20 @@ const createTransformer = (options: any) => {
       fileData: string,
       filename: Path,
       configString: string,
-      {instrument, watch}: TransformOptions,
+      {instrument}: TransformOptions,
     ): string {
-      return (
-        crypto
-          .createHash('md5')
-          .update(THIS_FILE)
-          .update('\0', 'utf8')
-          .update(fileData)
-          .update('\0', 'utf8')
-          .update(configString)
-          .update('\0', 'utf8')
-          // Don't use the in-memory cache in watch mode because the .babelrc
-          // file may be modified.
-          .update(getBabelRC(filename, {useCache: !watch}))
```

Comment:
Also, what's the idea behind moving `getBabelRC` into `createTransformer`? Just curious